tests/set/Makefile

```diff
@@ -0,0 +1,14 @@
+.include "../../share/mk/top.mk"
```

Comment:
Grab yourself copyright comments from other files, and stick your name at the top.